docs/template-cluster-cr.md

```diff
@@ -18,6 +18,7 @@ It supports the following flags:
   Use the flag once with a single value to create a cluster with one master node. For master node high availability,
   specify three distinct availability zones instead. This can be done by separating AZ names with comma or using the flag
   three times with a single AZ name.
+- `--credential` - AWS cloud credentials that point to the AWS account used to spin up the cluster resources. To get this info run against the Control Plane API `kubectl -n giantswarm get secret -oyaml | grep ORG_NAME -A2 | tail -n 1 | awk '{print $2}'` replacing `ORG_NAME` for the name of the organization selected.
```

Comment:
We dont want to add gs API logic here. It is just a templating tool so users need to provide the parameters. Ideally in official plugin this is deducted from org name (same as base domain or others).